src/Classes/EditControl.lua

```diff
@@ -157,6 +159,29 @@ function EditClass:Insert(text)
 	self:AddUndoState()
 end
 
+function EditClass:ZoomText(zoom)
+	if not self.allowZoom or not self.lineHeight then
+		return
+	end
+
+	local textHeight = self.lineHeight
+	if zoom == "+" then 
+		textHeight = textHeight + 1
+	elseif zoom == "-" then
+		textHeight = textHeight - 1
+	elseif zoom == "0" then
+		textHeight = self.defaultLineHeight
+	end
+
+	if textHeight < 1 then
```

Comment:
This could probably be higher.  Not sure how useful text this small is to anyone.

![image](https://user-images.githubusercontent.com/1209372/168730946-88e08dc0-eb62-4899-8cbb-be5b8799f234.png)